cyences_app_for_splunk/default/data/ui/views/cs_linux_reports.xml

```diff
@@ -170,29 +164,128 @@
   </row>
   <row>
     <panel>
-      <title>User List</title>
-      <input type="text" token="tkn_user_list_USERNAME">
-        <label>USERNAME</label>
+      <title>Linux Group Added/Updated/Removed</title>
```

Comment:
Can we have an additional panel to show Groups:
* like | stats values(user) as users, .... by host, group
* without changes, all users in single row.

Reply:
In group Panel you will get the same.